app.css

```diff
@@ -3269,60 +3269,96 @@ strong {
-  header .navbar {
-    width: 100%;
+header .notice-bar {
+  height: 70px;
+  background: #26A5D4;
+  color: #fff; }
+  header .notice-bar .notice-text {
+    padding-top: 26px;
+    font-size: 1.5rem;
+    font-weight: 300;
+    width: calc(100% - 200px);
+    margin: auto; }
+    header .notice-bar .notice-text a {
+      color: #fff;
+      text-decoration: underline; }
+  header .notice-bar #notice-bar-close {
     position: absolute;
-    left: 0;
-    top: 1rem;
-    z-index: 999; }
-    header .navbar .logo img {
-      max-height: 80px;
-      opacity: 1;
+    top: 0;
+    right: 2rem;
+    width: 70px;
+    height: 70px;
+    cursor: pointer; }
+    header .notice-bar #notice-bar-close:before, header .notice-bar #notice-bar-close:after {
+      height: 30px;
+      width: 2px;
+      background: #fff;
+      content: '';
+      position: absolute;
+      right: 3.3rem;
+      top: 18px;
       -webkit-transition: opacity 0.3s;
       -o-transition: opacity 0.3s;
       transition: opacity 0.3s; }
-      header .navbar .logo img:hover {
-        opacity: 0.7; }
-    header .navbar .navbar-collapse {
-      background: #fff;
-      padding-top: 5rem; }
-      header .navbar .navbar-collapse ul {
-        padding-left: 0;
-        list-style: none; }
-        header .navbar .navbar-collapse ul li {
-          display: inline-block;
-          padding: 0 1rem; }
-          header .navbar .navbar-collapse ul li a {
-            text-transform: uppercase;
-            font-weight: 900;
-            color: #333333;
-            -webkit-transition: color, 0.3s;
-            -o-transition: color, 0.3s;
-            transition: color, 0.3s; }
-            header .navbar .navbar-collapse ul li a:hover {
-              text-decoration: none;
-              color: #D85039; }
-          header .navbar .navbar-collapse ul li:last-child a {
-            color: #D85039;
-            border: solid 1px #D85039;
-            padding: 1rem 1rem;
-            -webkit-transition: background-color, 0.3s;
-            -o-transition: background-color, 0.3s;
-            transition: background-color, 0.3s; }
-            header .navbar .navbar-collapse ul li:last-child a:hover {
-              background: #D85039;
-              color: #fff;
-              border: solid 1px #D85039; }
-    header .navbar .navbar-toggle {
-      border: solid 1px #333333;
-      margin: 4rem 2rem; }
-      header .navbar .navbar-toggle .icon-bar {
-        background: #333333; }
+    header .notice-bar #notice-bar-close:before {
+      transform: rotate(-45deg); }
+    header .notice-bar #notice-bar-close:after {
+      transform: rotate(45deg); }
+    header .notice-bar #notice-bar-close:hover:before, header .notice-bar #notice-bar-close:hover:after {
+      opacity: .7; }
+
+header .navbar {
+  width: 100%;
+  margin-top: 1rem;
+  z-index: 999; }
+  header .navbar .logo img {
+    max-height: 80px;
+    opacity: 1;
+    -webkit-transition: opacity 0.3s;
```

Comment:
Maybe it is worth to consider adding the [postcss](http://postcss.org/) to the css generation instead of maintaining prefixes?

Reply:
:+1:

Reply:
Oh wait, it's transpiled `.scss`.

Reply:
You can add postcss to the pipeline like `SCSS|sass|postcss` when we have css generated.